modules/auxiliary/scanner/postgres/postgres_schemadump.rb

```diff
@@ -18,7 +18,6 @@ def initialize
       ),
       'Author' => ['theLightCosine'],
       'License' => MSF_LICENSE,
-      'SessionTypes' => %w[PostgreSQL]
```

Comment:
it still appears when you do `info -d` though right? I think that's how a majority of users would see it rather than looking at the module code itself

Reply:
I suppose that's true. Maybe I'm looking at it too much from a developer perspective were I'd be working with the module and just get my information there. Where as you say, users would probably get it from console.

Reply:
Dean's current pattern looks good to me :+1: